### DISCUSSION TITLE
Using `updateEdge()` with Composition API doesn't immediately render update

### DISCUSSION THREAD

**Discussion by Modjular** [bcakmakoglu/vue-flow#294] [Q&A] [answered]
Here's how I'm using `updateEdge` in my SFC
```typescript
onEdgeUpdate(({ edge, connection }) => updateEdge(edge, connection))
```
And my `VueFlow` config
```Vue
<template>
  <VueFlow
    v-model="elements"
    :edges-updatable="true"
    :fit-view-on-init="true"
    :connection-mode="ConnectionMode.Strict"
  >
...
```

When dragging the edge to a new handle, it updates the store, but the changes don't appear until I cause a re-paint, such as dragging a node.

Am I missing a setting in the config?

**Comment by bcakmakoglu** [comment, marked as answer]
No, that seems to be a bug.

For now you can just use the way demonstrated in the examples.

```js
const onEdgeUpdate = ({ edge, connection }) => {
  elements.value = updateEdge(edge, connection, elements.value)
}
```

**Reply by bcakmakoglu** [reply]
See #296 for a coming fix.
